src/app/code/community/FireGento/GridControl/Model/Processor.php

```diff
@@ -110,9 +110,15 @@ protected function _addAction($params)
                 if (strpos((string) $attribute, '::') !== false) {
                     list($_module, $_method) = explode('::', (string) $attribute);
                     $_module = Mage::getSingleton($_module);
+                    if ($_args = strstr($_method, '(')) {
+                        $_method = str_replace($_args, '', $_method);
+                        $_args = array_map('trim', explode(',', str_replace(array('(', ')', '\'', '"'), '', $_args)));
```

Comment:
I prefer the implementation as documented because it's more compact https://github.com/magento-hackathon/GridControl/pull/19. If you feel strongly about the more-magento-y syntax I guess we can update it though.

It would read like this...

```xml
<?xml version="1.0"?>
<gridcontrol>
    <grids>
        <productGrid>
           <manufacturer>
                <add>
                    <header>Brand</header>
                    <type>options</type>
                    <index>manufacturer</index>
                    <options>mpchadwick_gridconfig/utility::getDropdownAttributeLabelOptionArray
                        <args>
                            <arg1>manufacturer</arg1>
                        </args>
                    </options>
                </add>
                <after>name</after>
            </manufacturer>
        </productGrid>
    </grids>
</gridcontrol>
```